datadog/resource_datadog_dashboard.go

```diff
@@ -4187,6 +4205,9 @@ func getScatterplotRequestSchema() map[string]*schema.Schema {
 		"process_query":  getProcessQuerySchema(),
 		"rum_query":      getApmLogNetworkRumSecurityAuditQuerySchema(),
 		"security_query": getApmLogNetworkRumSecurityAuditQuerySchema(),
+		// "query" and "formula" go together
+		"query":   getFormulaQuerySchema(),
+		"formula": getScatterplotFormulaSchema(),
```

Comment:
Should have those in their own schema, to reuse in the `scatterplot_table` attribute

Reply:
Resolved in [e8e0](https://github.com/DataDog/terraform-provider-datadog/pull/1275/commits/9ecadcc1d889605b30f5de449d7098231ac2e8e0).